.github/workflows/mac.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       fail-fast: true
       matrix:
         include:
-          - platform: "macos-latest"
+          - platform: "macos-13"
             architecture: "x64"
           # macOS-14 is a M1 ARM64 MacOS runner: https://github.blog/changelog/2024-01-30-github-actions-introducing-the-new-m1-macos-runner-available-to-open-source/
           - platform: "macOS-14"
```

Comment:
Should we change this to `latest`? otherwise it is probable that we do not update to the newer ones.

Reply:
I don't know. The -latest is also a moving target that has switched between x64 and arm64, which is annoying. It is weird that they don't provide 2 separate keywords, one for the mac version and the other one for the architecture. But I believe Apple won't support x64 anymore, so perhaps at some point that will be arm64 only.

Reply:
Oh, yes. I think that the new models are all arm. The question is how long they will support the old ones.